aws/main.tf

```diff
@@ -209,7 +209,8 @@ module "ec2_database" {
           postgres_password = var.blockscout_settings["postgres_password"]
           postgres_user     = var.blockscout_settings["postgres_user"]
         }
-      )
+      ),
+      indexer_json_config       = "TODO: not needed here, maybe the var can be marked as optional? in the template or a separate template introduced"
```

Comment:
Added here just because the template was complaining about a missing var. We probably need to add a separate template for the db, or use optional vars if that option exist. Didnt have time to investigate further today, will do it tomorrow.